lib/chef/client.rb

```diff
@@ -537,6 +538,19 @@ def has_admin_privileges?
       Chef::ReservedNames::Win32::Security.has_admin_privileges?
     end
 
+    def wrap_exceptions(converge_error, audit_error)
+      if audit_error && !(audit_error.is_a?(Chef::Exceptions::AuditsFailed) && Chef::Config[:audit_as_warning])
```

Comment:
audit failure is a bit ambiguous: was there an error during the audit phase or did some control not pass?